### DISCUSSION TITLE
Github Learning Lab

### DISCUSSION THREAD

**Discussion by ericbarrionuevo** [community/community#30311] [Other Feature Feedback, Questions, & Ideas] [answered]
hello, git learning will definitely close on september 1?? Cheers

**Comment by queenofcorgis** [comment, marked as answer]
That is correct, Learning Lab will shut down on September 1, 2022. However, we recently launched GitHub Skills, learn more about the shutdown and GitHub Skills [here](https://github.com/orgs/community/discussions/28323).

**Reply by ZhaoKunLong** [reply]
@queenofcorgis   Hello! have a nice day! Before I think the GitHub Skills will better, But now it's awful.
This is really a wrong decision. I tried the latest GitHub skills. It is not as friendly as the previous GitHub learning lab.

1. No learning progress bar
2. The course is not grouped
3. The course is not gradual
4. Too few courses
6. There is no interaction in the course.

It simply put a few repos and then left it alone, it's really awful.
Is there any way to get the learning lab back? Even if it is a new domain name.

**Reply by queenofcorgis** [reply]
Hi there @ZhaoKunLong 👋🏼 

Thank you so much for this feedback, I'm going to pass it on to our GitHub Skills team. Unfortunately, Learning Lab has been deprecated as it is now September 1st. GitHub Skills will continue to evolve especially as fine folks like you continue to leave feedback.

**Reply by ZhaoKunLong** [reply]
@queenofcorgis  Thanks for your reply. And If it is a foregone conclusion, 
I can only pin my hope on the new GitHub skills. Where can I provide feedback continuously? 

What I need most is to show the course progress. It is possible that a complete course contains 3 lessons, and I only learned the second. We hope to have such a feature.
The second is whether we can add more courses about GitHub actions.

GitHub learning is really great. I recommended GitHub learning to my classmates before. Now they are all learning with it. 
It's grateful to your guys developed this site.
